Meridian59.DebugUI/CustomDataGridColumns/TagColumn.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace Meridian59.DebugUI.CustomDataGridColumns
+{
+    class TagCell : DataGridViewTextBoxCell
```

Comment:
Just a placeholder. I want to have a custom column for the "Tag/Blakserv DataType" column in the ObjectEditor form.